src/js/api/auth/register.js

```diff
@@ -1,59 +1,29 @@
-import { apiUrl } from '../constants.js';
+import { apiPath } from '../constants.js';
 
-const action = 'users';
-const method = 'POST';
```

Comment:
This is fine being uppercase too: https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods